super/pom.xml

```diff
@@ -127,41 +130,12 @@
     <cucumber.version>5.6.0</cucumber.version>
     <sun.httpserver.version>20070405</sun.httpserver.version>
     <postgresql.version>42.2.12</postgresql.version>
+    <javax.xml.soap-api.version>1.4.0</javax.xml.soap-api.version>
+    <kafka.version>2.4.0</kafka.version>
+    <avro.version>1.9.1</avro.version>
+    <spring.kafka.version>2.4.4.RELEASE</spring.kafka.version>
```

Comment:
Adapt the super pom of the applications so these values are not duplicated

Reply:
Will do this when refactoring the Kafka configuration in osgp-webservice-client.